src/Analyzer.Core.UnitTests/TemplateAnalyzerTests.cs

```diff
@@ -48,10 +48,10 @@ private string GenerateResource(string resourceProperties, string resourceType)
             return string.Format(@"
                     {{
                       ""apiVersion"": ""2018-02-01"",
-                      ""name"": ""resourceName"",
+                      ""name"": ""{2}"",
                       ""type"": ""{1}"",
                       ""properties"": {0}
-                    }}", resourceProperties, resourceType);
+                    }}", resourceProperties, resourceType, resourceName);
         }
```

Comment:
I believe it's because I do a match with resource name and type, I needed the names to be different. It's expecting name and resource type combinations to be unique in a template.